### DISCUSSION TITLE
How to use Wallbash Modes,Notification Action and Game Launcher

### DISCUSSION THREAD

**Discussion by AKA-Akhil** [prasanthrangan/hyprdots#1449] [Q&A] [answered]
Hey i am new here so can anyone tell me the key bindings for 
1)Wallbash Modes
2)Notification Action
3)Game Launcher

**Comment by kRHYME7** [comment]
click the keyboard glyph in waybar. It will show a keybindhint list. you can search and execute it directly from there.


 also the keybindings are listed in README

**Reply by AKA-Akhil** [reply]
yeah but i cldnt find them are they liisted under a different name?

**Reply by kRHYME7** [reply, marked as answer]
You can set it yourself, IIRC there is no keybind for that.


The script name is ` gamelauncher.sh ` .

This is how to add key binds in Hyprland https://wiki.hyprland.org/Configuring/Binds/

Edit your ` ~/.config/hypr/userprefs.conf ` then add your preferred keybind. (read the wiki first )

NOte: Don't worry the keybind hint is auto generated so it should reflect. 


TIP. ` gamelauncher MODE` got modes from 1 to 5 so test which layout you preferred. 

eg ` gamelauncher 2 `

**Reply by Legion2016** [reply]
Hello. I don't have a game launcher script

**Comment by kRHYME7** [comment]
>  2)Notification Action

We don't have that yet, but there is a neat way, notifications are in the tooltip

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/284c76db-c5af-4668-9bff-8c35f16d83a7)

**Comment by kRHYME7** [comment]
>  1)Wallbash Modes

` wallbashtoggle.sh -m `  
shows a MENU

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/2cf28324-579a-4bac-982b-d8ecb80a8abd)

theme: Will follow the theming scheme if available
auto: Auto detect if light/dark mode 
dark: force dark mode no matter what
light: force light mode no matter what 


I assume you already knew how to add keybinds so here's the key bind to activate wallbashtoggle.sh 
` ctrl + Super + d `

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/b14ea7f0-1203-4407-997b-27e046c65a8c)

**Reply by AKA-Akhil** [reply]
yeah thanks!
